src/Common/src/System/Data/Locale/Locale.cs

```diff
@@ -1,58 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Collections.Concurrent;
-using System.Data.Common;
-using System.Globalization;
-using System.Text;
-
-namespace System.Data
-{
-    internal static class Locale
-    {
-        // Maps between LCIDs and LocaleName+AnsiCodePages+Encodings
-        private static readonly ConcurrentDictionary<int, Tuple<string, int, Encoding>> s_cachedEncodings = new ConcurrentDictionary<int, Tuple<string, int, Encoding>>();
-
-        static Locale()
-        {
-            Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);            
```

Comment:
I was wondering if we'd need to find another place to register the `CodePagesEncodingProvider` provider, but it's already being registered in [`TdsParser`](https://github.com/dotnet/corefx/blob/562942307a5d2c771d6ff9c14a87a13e40c59b26/src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParser.cs#L34), the only class in `System.Data.SqlClient` where `Encoding.GetEncoding(int)` is called.